scripts/install2

```diff
@@ -79,10 +79,20 @@ fi
 
 # Get tools
 if ! [ "$(which nodejs)" ]; then
-    curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
-    sudo apt-get install nodejs -y
-fi
 
+#check for armv6 and install it manually instead
+    if $(uname -m | grep -Eq ^armv6); then
+pwd
+        wget -O /tmp/node-v6.11.0-linux-armv6l.tar.gz https://nodejs.org/dist/v6.11.0/node-v6.11.0-linux-armv6l.tar.gz
+        sudo tar xfz /tmp/node-v6.11.0-linux-armv6l.tar.gz --strip 1 -C /
+	rm -rf /tmp/node-v6.11.0-linux-armv6l.tar.gz
+        sudo ln -s /bin/node /bin/nodejs
```

Comment:
it is needed beucase the install script looks for nodejs

`if ! [ "$(which nodejs)" ]; then`